chef/data_bags/crowbar/migrate/nova_dashboard/026_use_keystone_session_timeout.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  keystone_instance = a['keystone_instance']
+  proposal_obj = ProposalObject.find_proposal("keystone", keystone_instance)
+  keystone_timeout = proposal_obj["attributes"]["keystone"]["token_expiration"]/60
```

Comment:
> hrm, okay. I would have dealt with it here by falling back on some value, but I'm fine either way.

@vuntz That's what I meant with "Still the migration should handle it as well though". We should assume a keystone time out of 3600 sec in that case. As that was the hardcoded default before we made the setting available in keystone.